contracts/factory/NexusAccountFactory.sol

```diff
@@ -58,6 +58,8 @@ contract NexusAccountFactory is Stakeable, INexusFactory {
         if (!alreadyDeployed) {
             INexus(account).initializeAccount(initData);
             emit AccountCreated(account, initData, salt);
+        } else if (msg.value > 0) {
+            revert AccountAlreadyDeployed(account);
```

Comment:
can we have negative test case for this how entrypoint behaves.

Reply:
by sending a userop

Reply:
done